docs/en/latest/mtls.md

```diff
@@ -103,7 +103,7 @@ import sys
 # sudo pip install requests
 import requests
 
-if len(sys.argv) <= 4:
+if len(sys.argv) < 4:
```

Comment:
I am afraid this is necessary.

Reply:
It's okay, I think the script is being adapted to be used in storing server certs and keys in APISIX as well, along with mTLS client ca information. It's a minor change and if it ends up helping people, I think it's okay to be merged : )